pandas/core/ops/array_ops.py

```diff
@@ -250,7 +257,10 @@ def comparison_op(
         op_name = f"__{op.__name__}__"
         method = getattr(lvalues, op_name)
         with np.errstate(all="ignore"):
-            res_values = method(rvalues)
+            res_values = na_arithmetic_op(lvalues, rvalues, op, str_rep, is_cmp=True)
```

Comment:
why is thi not handled in na_arithmetic_op? seems odd to handle here

Reply:
sure